code/part-two/processor/utils/addressing.js

```diff
@@ -37,7 +37,7 @@ getAddress.offer = ownerKey => {
     return NAMESPACE +
       TYPE_PREFIXES.OFFER +
       ownerHash +
-      hash(offeredAddresses.join(''), 54);
+      hash(offeredAddresses.sort().join(''), 54);
```

Comment:
yesss, I kept thinking "WE CAN'T FORGET TO SORT THE ADDRESSES" all of last week, so I'm glad you remembered to put it in!